src/test/osd/CMakeLists.txt

```diff
@@ -114,3 +114,12 @@ add_ceph_unittest(unittest_mclock_client_queue)
 target_link_libraries(unittest_mclock_client_queue
   global osd dmclock os
 )
+
+# unittest_mclock_pool_queue
+add_executable(unittest_mclock_pool_queue
+  TestMClockPoolQueue.cc
+)
+add_ceph_unittest(unittest_mclock_pool_queue)
+target_link_libraries(unittest_mclock_pool_queue
+  global osd dmclock os
```

Comment:
As you say, the queue of TestMClockPoolQueue.cc does not intuitively use the osd and os libraries. 
But since the request consists of PGSnapTrim, PGScrub, PGRecovery, and OpRequestRef, we need an osd library to compile, and when we use osd, Connected os are also needed together.